### PR TITLE
NPM-ify

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
 - Prepare Neue for [publishing on NPM](https://www.npmjs.com/package/dosomething-neue). This will help to simplify our front-end architecture as we split things out into more packages.

For review: @DoSomething/front-end

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,25 +1,11 @@
...
     "dist/neue.css",
-    "dist/neue.min.js"
+    "dist/neue.js"
   ],
   "dependencies": {
     "jquery": "^1.7",
     "susy": "2.2.2"
-  },
-  "devDependencies": {
-    "jquery": "1.8.3",
```
Comment (sbsmith86) [inline bower.json]: @DFurnes Why do we remove jquery as a dependency? How does it get added?

Comment (sbsmith86) [inline bower.json]: nevermind!

**Comment by sbsmith86** [comment]
Looks good to me :+1: maybe we can have a fender sync sometime to go over all the updates you are making? Can't say I understand all of it, but it looks like great stuff!

**Comment by DFurnes** [comment]
Definitely!

**Pull request merged**